src/fiatconnect/constants.ts

```diff
@@ -0,0 +1,25 @@
+import { FiatAccountSchema } from '@fiatconnect/fiatconnect-types'
+import { LocalCurrencyCode } from 'src/localCurrency/consts'
+
+export const CURRENCIES_WITH_FEE_DISCLAIMER: Partial<
```

Comment:
the AC for the ticket says this list should be the list of *countries* not currencies. Any reason why you did currency instead?

Reply:
I initially interpreted the 2nd AC to mean we just use the currency codes of those countries, but I've updated the code to rely on countries.